pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -11351,26 +11351,14 @@ func TestReconcile_verifyResolvedPipeline_Success(t *testing.T) {
 	ctx := context.Background()
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
+	// Set up a fake registry to push an image to.
+	s := httptest.NewServer(registry.New())
+	defer s.Close()
+	u, err := url.Parse(s.URL)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
use oci bundle for pipelinerun_test and taskrun_test, since we probably cannot use remote resolution at these tests, I cannot find any tests in these functions.